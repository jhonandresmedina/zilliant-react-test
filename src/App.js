import '../node_modules/react-md/dist/react-md.deep_orange-blue.min.css';

//@vendor
import React from 'react';

//@components
import Layout from './github/components/Layout';
import Repos from './github/components/Repos';

const App = () => {
    return (
        <Layout>
            <Repos />
        </Layout>
    );
};

export default App;
