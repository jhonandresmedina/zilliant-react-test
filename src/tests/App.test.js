//@vendor
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

//@components
import App from '../App';

Enzyme.configure({ adapter: new Adapter() });

describe('App test', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<App />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
