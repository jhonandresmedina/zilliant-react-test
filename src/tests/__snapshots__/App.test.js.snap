// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test renders correctly 1`] = `
<Connect(Layout)>
  <Connect(Repos) />
</Connect(Layout)>
`;
